enhancements/20220823-harvester-plugin.md

```diff
@@ -0,0 +1,81 @@
+# Harvester Plugin
+
+## Summary
+The core code and components of rancher and harvester are very similar. Harvester hopes to easily reuse the common components of rancher ui and the steve architecture, without the need to push the code to the rancher dashboard repo. The purpose is to hope that the harvester can be released independently and not with rancher bindings.
+
+
+### Related Issues
+
+https://github.com/rancher/dashboard/issues/4109
+
+## Motivation
+It is hoped that the harvester can utilize the core code and common components of rancher, and can build the release version independently, and the multi-cluster harvester ui can also be easily integrated with the rancher.
+
+### Goals
+
+1. Harvester and Rancher as two different projects, hope to be developed and released independently.
+2. Hope harvester can be managed by rancher.
+3. In harvester multi-cluster, it is hoped that each imported harvester cluster can configure the version of the plugin and whether the access is an offline plugin or an online plugin
```

Comment:
Yes, online access means that the user environment needs to be able to access the external Internet, offline access has no such requirement.    
The user's intranet deployment of harvester may not be able to access the external Internet, then it will not be able to access such a network resource  `https://releases.rancher.com/harvester-ui/plugin/harvester-latest/harvester-latest.umd. min.js` , so we need to put this resource inside the harvester so that users can access it directly `https://172.16.0.148/v1/harvester/plugin-assets/`

Reply:
Thanks for the detailed explanation! I have modified the code suggestion accordingly~